src/components/AssetDownload/AssetDownloadArtist.tsx

```diff
@@ -17,20 +19,17 @@ const AssetDownloadArtist = ({
   const { t } = useTranslation("page-assets")
   return (
     <Flex
-      mb="4"
-      border="1px"
-      borderTop="none"
-      borderColor="white700"
-      py="2"
-      px="4"
-      borderRadius="0 0 4px 4px"
+      className={cn(
+        "mb-4 border border-t-0 border-[#e5e5e5]",
```

Comment:
Will remove these colors and leave the border default for now
```suggestion
        "mb-4 border border-t-0",
```